apps/css-workshop/src/components/Menu.astro

```diff
@@ -0,0 +1,68 @@
+---
+type Props = {
+  isActive?: boolean;
+  checked?: boolean;
+  isLarge?: boolean;
+  disabled?: boolean;
+  isSkeleton?: boolean;
+  iconStyle?: string;
+  iconClassName?: string;
+} & astroHTML.JSX.HTMLAttributes;
+
+const {
+  class: className,
+  isActive,
+  checked,
+  isLarge,
+  disabled,
+  isSkeleton,
+  iconStyle,
+  iconClassName,
+  ...props
+} = Astro.props;
+---
+
+<li
+  class:list={[
+    { 'iui-list-item': !isSkeleton },
+    { 'iui-menu-item': isSkeleton },
+    { 'iui-active': isActive },
+    { 'iui-disabled': disabled },
+    { 'iui-menu-item-skeleton': isSkeleton },
+    { 'iui-large': isLarge },
+    className,
+  ]}
+  aria-checked={checked}
```

Comment:
`aria-checked` is not valid on list items; see [MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-checked#associated_roles).

Reply:
I have removed all `aria-checked` from the items.